### ISSUE TITLE
1.1.0 ignores trailing white spaces on lines without text

### ISSUE THREAD

**Issue by jby** [rodjek/puppet-lint#359]
This file is tested without error (space made visible by brackets):
```puppet
# Class foo
class foo {
[ ]
}
```
If the file looks like this, the error is reported (space made visible by brackets):
```puppet
# Class foo
class foo {
  include foo[ ]
}
```
